src/patternfly/components/Nav/nav.scss

```diff
@@ -169,37 +204,75 @@
   --#{$nav}__item--RowGap--row-offset: calc(var(--#{$nav}__list--RowGap) / 2 * -1); // row gap offset calcs 1/2 height of row gap
   --#{$nav}__link--clickable-inset--Block: var(--#{$nav}__item--RowGap--row-offset); // set link's clickable area offset based on 1/2 row gap
   --#{$nav}__button--RowGap--row-offset: calc(var(--#{$nav}__item--RowGap) * -1); // match extra bottom paddings applied to expanded nav_links
+
+  visibility: visible;
 }
 
 // - Nav list
 .#{$nav}__list {
   row-gap: var(--#{$nav}__list--RowGap);
   column-gap: var(--#{$nav}__list--ColumnGap);
+  min-height: 0;
+  opacity: 1;
+  transition-timing-function: var(--#{$nav}__list--TransitionTimingFunction--opacity);
+  transition-duration: var(--#{$nav}__list--TransitionDuration--opacity);
+  transition-property: opacity;
+  transition-behavior: allow-discrete;
+
+  @starting-style {
+    opacity: 0;
+  }
```

Comment:
You shouldn't need this, either. It's already 0 when the subnav is hidden https://github.com/patternfly/patternfly/blob/b635f763cec40fb2cf4ae5f84ec04f31a83fde6d/src/patternfly/components/Nav/nav.scss#L189-L197